subprojects/inline/inline.gradle

```diff
@@ -0,0 +1,18 @@
+description = "Mockito preconfigured inline mock maker"
```

Comment:
Yes until we find a better way to activate final mocking either explicitly or automatically.